go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/conductorone/baton-celigo
 go 1.22
 
 require (
-	github.com/conductorone/baton-sdk v0.1.23
+	github.com/conductorone/baton-sdk v0.1.25
```

Comment:
Needs to be bumped to v0.1.26 to fix this bug: https://github.com/ConductorOne/baton-sdk/commit/5cced7c68160f2294533b83fc467ece76b3105c6